src/main/java/de/tum/in/www1/artemis/domain/enumeration/CourseInformationSharingConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package de.tum.in.www1.artemis.domain.enumeration;
+
+/**
+ * NOTE: Ordinal mapping in property of Course.java do NOT change the order of the existing values.
+ */
+public enum CourseInformationSharingConfiguration {
```

Comment:
Is it absolutely clear that we will not have any other messaging in the near future? E.G. only messages to tutors but not other students?
Otherwise we'll potentially have an exponential blow up of values here.

Additionally I think two booleans might be easier in the rest of the code. For example in the html files where you currently call methods. Sometimes they have the problem that they are called over and over again, did you check that this is not the case here?

Reply:
The decision to use enums was made in discussion with @krusche.